src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int height;
+    private int width;
+    private String color = getColor();
+
+    public Rectangle(int height, int width, String color) {
+        this.height = height;
+        this.width = width;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (height * width);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure rectangle, area: " + areaCalculator()
+                + ", height " + height + ", width " + width
+                + ", color: " + color);
+    }
+}
```

Comment:
Class names should not be verbs, but rather nouns or noun phrases. Consider renaming the class to something like 'RectangularFigure' if the class represents a figure with a rectangular shape.